### PR TITLE
feat: added issue-template

### PR DESCRIPTION
**Pull request by iamayushdas**
fixed: #182 (Added issue template)

### CODE REVIEW

**Comment by iamayushdas** [comment]
cc @Yiyiyimu

File: .github/workflows/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Screenshots
+<!-- Add screenshots to help explain your problem if applicable. -->
+
+## Environment
+
+* apisix version (cmd: `apisix version`):
+* OS (cmd: `uname -a`):
+* OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
+* apisix-docker version, you have:
```
Comment (Yiyiyimu) [inline .github/workflows/ISSUE_TEMPLATE/bug-report.md:24]: apisix docker version is enough

File: .github/workflows/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Visit Apache APISIX
+    url: https://github.com/apache/apisix
+    about: Ask questions or discuss with other community members about Apache APISIX
```
Comment (Yiyiyimu) [inline .github/workflows/ISSUE_TEMPLATE/config.yml:5]: newline missing

**Review by Yiyiyimu — CHANGES_REQUESTED** [review]

File: .github/workflows/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (iamayushdas) [reply .github/workflows/ISSUE_TEMPLATE/config.yml:5]: okay sir

**Review by Yiyiyimu — APPROVED** [review]

File: .github/workflows/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## How to Reproduce
+<!-- Add how you reproduce that error. -->
+
+## Screenshots
+<!-- Add screenshots to help explain your problem if applicable. -->
+
+## Environment
+
+* apisix-docker version, you have:
```
Comment (Yiyiyimu) [inline .github/workflows/ISSUE_TEMPLATE/bug-report.md:21]:
```suggestion
* apisix-docker version:
```

**Review by tokers — APPROVED** [review]

**Pull request merged**